spec/models/repository_spec.rb

```diff
@@ -603,8 +602,8 @@ def get_url(repo, tag)
     it "groups tags as expected" do
       tags = repo.groupped_tags
       expect(tags.size).to eq 2
-      expect(tags.flatten.map(&:name).uniq).to eq [tag1.name, tag2.name, tag3.name]
-      expect(tags.flatten.map(&:digest).uniq).to eq ["1234", "5678"]
+      expect(tags.flatten.map(&:name).uniq).to include(tag1.name, tag2.name, tag3.name)
+      expect(tags.flatten.map(&:digest).uniq).to include("1234", "5678")
```

Comment:
Does include guarantee order? Maybe another side effect due to the sort thing? If we are going with `include`, then we should also check the length to be sure that it's exactly the same content.

Reply:
This was introduced before the other one. Overall, there's just something off with assumptions we made on time data.

I agree that we should check the length. Let me fix that.

Reply:
> I agree that we should check the length. Let me fix that.

👍